ui/pages/confirm-transaction-base/transaction-alerts/transaction-alerts.test.js

```diff
@@ -56,4 +59,37 @@ describe('LowPriorityMessage', () => {
       document.getElementsByClassName('actionable-message--warning'),
     ).toHaveLength(0);
   });
+
+  it('should not show insufficient balance message if transaction value is less than balance', () => {
+    render({
+      transaction: { userFeeLevel: 'high', txParams: { value: '0x64' } },
+    });
+    expect(screen.queryByText('Insufficient funds.')).not.toBeInTheDocument();
+  });
+
+  it('should show insufficient balance message if transaction value is more than balance', () => {
+    render({
+      transaction: { userFeeLevel: 'high', txParams: { value: '0x5208' } },
+    });
+    expect(screen.queryByText('Insufficient funds.')).toBeInTheDocument();
+  });
+
+  it('should show pending transaction message if there are >= 1 pending transactions', () => {
+    render(undefined, {
```

Comment:
What do you think about having `render` take an object rather than two positional arguments? Then instead of having to pass `undefined` her you could call this like the following:

``` javascript
render({
  state: {
    currentNetworkTxList: [...],
    ...
  }
})
```

I think it would make it more obvious what each thing represents as well.